packages/expo-modules-autolinking/src/platforms/ios.ts

```diff
@@ -56,16 +56,19 @@ async function generatePackageListFileContentAsync(
   const pods = modulesToProvide.map((module) => module.podName);
   const classNames = [].concat(...modulesToProvide.map((module) => module.modulesClassNames));
 
-  return `// Automatically generated by expo-modules-autolinking.
-import ExpoModulesCore
-
-${pods.map((podName) => `import ${podName}`).join('\n')}
+  return `/**
+ * Automatically generated by expo-modules-autolinking.
+ *
+ * This autogenerated class provides a list of classes of native Expo modules,
+ * but only these that are written in Swift and use the new API for creating Expo modules.
+ */
 
+import ExpoModulesCore
+${pods.map((podName) => `import ${podName}\n`).join('')}
 @objc(${className})
 public class ${className}: ModulesProvider {
-  public override func exportedModules() -> [AnyModule.Type] {
-    return [
-      ${classNames.map((className) => `${className}.self`).join(',\n      ')}
+  public override func getModuleClasses() -> [AnyModule.Type] {
```

Comment:
Swift's naming convention tends to leave out `get` (however, Java and JS's conventions should include `get`). `moduleClasses` would follow the Swift conventions more closely.

https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage